examples/storage/couchdb/couchdb-petset.yaml

```diff
@@ -0,0 +1,114 @@
+apiVersion: "apps/v1alpha1"
+kind: PetSet
+metadata:
+  name: kubetest001
+  namespace: default
+spec:
+  serviceName: kubetest001-backend
+  replicas: 3
+  template:
+    metadata:
+      annotations:
+        pod.alpha.kubernetes.io/initialized: "true"
+        pod.alpha.kubernetes.io/init-containers: '[
+            {
+            "name": "configer-couch",
+            "image": "glyndwr/kube-init-couchdb",
+            "command": ["/bin/bash", "-c", "/./configer_couch.sh"],
```

Comment:
So this is a hangover from when all of the kube specific configuration happens in the one container that included couchdb.  So the entrypoint was not set to `/`